scripts/initialize/download_data.sh

```diff
@@ -11,9 +11,13 @@ synapse get syn4976369
 mv EB++AdjustPANCAN_IlluminaHiSeq_RNASeqV2.geneExp.tsv \
 data/raw/pancan_normalized_rnaseq.tsv
 
-# Clinical sample freeze info (downloaded 11 Jan 2017)
+# Clinical sample freeze info (downloaded 7 March 2017)
 # Google doc located here
 # https://docs.google.com/spreadsheets/d/1Z1H3mXdO_sk9nc0v8df7VNS_XzXiu6vKGJRbK1qYoh4/edit?usp=gmail
+# Move to `data/raw/sampleset_freeze_version4_modify.csv`
+# Modify sample freeze internally:
+# Replace '01' to '02' in the tumor code following four samples:
```

Comment:
in the sample freeze, the tumor barcodes are listed as `01` which indicates a primary tumor. The RNAseq data actually list these samples as `02` which indicates a recurrent tumor (from the same individual). For example, `TCGA-06-0152-02` is the recurrent tumor for patient `TCGA-06-0152` while `TCGA-06-0152-01` is the primary.

These samples only have recurrent tumors in the RNAseq data, but have mutation data for both.